fw-child/resources/js/child-functions.js

```diff
@@ -609,47 +609,77 @@ const $ = jQuery;
     
       $('.scroll-card').each(function () {
         const card = this;
-    
+        const isLastCard = $(card).closest('.fw-query-item').is(':last-child');
+
+         // Set pointer-events to none at the beginning for every card
+        $(card).css('pointer-events', 'none');
+        
+        // Define common properties
+        let fromToProps = { 
```

Comment:
Since the variable `fromToProps` is not redefined, we should use `const` (even if the attributes of the variable are changed).

```suggestion
        const fromToProps = { 
```